### ISSUE TITLE
"Link to search" is not functioning in Metadata Display facet in Dashboard

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#1199]
When I click on "link to search" in thee metadata display options and click save, the selected fields are not linked to search.  Current links active before upgrade are still functional. 
<img width="1440" alt="Screen Shot 2021-11-02 at 9 46 02 PM" src="https://user-images.githubusercontent.com/45043014/139999583-9f942e78-b285-4f9f-a305-d8e4f2750276.png">

**Comment by eliotjordan** [comment]
We forgot to run `LinkToSearchMigration` on prod.

**Comment by kelea99** [comment]
Tested "link to search " in prod. , @eliotjordan  What i found:
1. the items I tried linking yesterday are now linked in DPUL
2. the items I have tried linking today (click linke to search and save). are not linked.

**Comment by kelea99** [comment]
Reopening this. The links now show but the search functionality of them does not. I click on a subject heading and still get the all search, versus the search of that specific subject.

**Comment by hackartisan** [comment]
link is going to

`/piranesi/catalog?f%5Bexhibit_piranesi_readonly_author_ssim%5D%5B%5D=Vasi%2C+Giuseppe%2C+1710-1782`

instead of 

`piranesi/catalog?f%5Breadonly_author_ssim%5D%5B%5D=Vasi%2C+Giuseppe%2C+1710-1782&q=&search_field=all_fields`

The facet prefix seems to be the issue here. It works with or without the search_field param.

**Comment by kelea99** [comment]
Reopening this.  Link  to search is not functioning for creator in Libretti: https://dpul.princeton.edu/russian-opera-libretti/catalog?utf8=%E2%9C%93&f%5Bexhibit_russian-opera-libretti_readonly_subject_ssim%5D%5B%5D=Operas%E2%80%94Librettos&exhibit_id=russian-opera-libretti&search_field=all_fields&q=

**Comment by hackartisan** [comment]
@kelea99 sorry I didn't communicate the resolution here very well. Any exhibit whose metadata configuration was updated between the upgrade and the merge / deploy of #1224 can be repaired by going to the dashboard's metadata configuration page and clicking "save". That will refresh the link to search field to have the right value. I just did it for this Russian Opera collection. Let me know if I can help with any others.
